categorize_adjectives/libraries/bert_embeddings.py

```diff
@@ -0,0 +1,153 @@
+"""Contains function for calculating BERT embeddings"""
```

Comment:
Can you combine this file with bert.py in previous PR?
nit: categorize_adjectives/utils might be good naming than categorize_adjectives/libraries

Reply:
For utils, I don't think so as utils is generally used for general and trivial function. This is a complete class and a non trivial function.